bower.json

```diff
@@ -3,20 +3,18 @@
   "version": "0.0.0",
   "dependencies": {
     "angular-bootstrap": "~0.10.0",
-    "angular-resource": "~1.2.13",
-    "angular-route": "~1.2.13",
+    "angular-resource": "~1.6.7",
+    "angular-route": "~1.6.7",
     "angular-ui-date": "~0.0.3",
-    "angular": "~1.2.13",
+    "angular": "~1.6.7",
```

Comment:
1.6.6 didnt work for @moe but I'm assuming all the libraries were installed with 1.6.7?

Reply:
yeah i think it should work. @Moe, can you verify on your local? :)

Reply:
it works